### PR TITLE
Devel branch

### PR DESCRIPTION
**Pull request by danfossi**
* Devel branch with Bootstrap 4

### CODE REVIEW

**Comment by serhack** [comment]
What can't you edit on application/controller/shop.php ?

**Comment by danfossi** [comment]
Having some issues with the function: public function generate_nav() on application/controller/Admin.php

File: application/controllers/Admin.php [hunk 1]
```diff
@@ -99,12 +99,12 @@ public function generate_nav()
...
             $links .= '<li';
             if (uri_string() == 'admin' . $entry['panel'] || uri_string() == 'admin/edit' . $entry['panel']) {
-                $links .= ' class="active" ';
+                $links .= ' class="active nav-item" ';
                 $self = $entry;
                 $heading = $entry['heading'];
                 $panel_url = $self['panel'];
             }
-            $links .= '>' . anchor('admin' . $entry['panel'], $entry['title']) . '</li>';
+	    $links .= '>' . anchor('admin' . $entry['panel'],  $entry['title'], array('class' => 'nav-link')) . '</li>';
```
Comment (danfossi) [inline application/controllers/Admin.php]: * added "nav-link" class

File: application/controllers/Admin.php [hunk 2]
```diff
@@ -113,10 +113,10 @@ public function generate_nav()
                 <li class="col-xs-3"><h4>' . $self['heading'] . '</h4></li>
                 <li class="col-xs-1">';
 
-        if ($panel_url !== '/logs' AND $panel_url !== '/autorun' ) $nav .= " " . anchor('admin/edit' . $panel_url, '<span class="glyphicon glyphicon-wrench"></span>', 'class="btn btn-xs btn-primary"');
+		if ($panel_url !== '/logs' AND $panel_url !== '/autorun') $nav .= " " . anchor('admin/edit' . $panel_url, 'Edit', array('class' => 'nav-link'));
```
Comment (danfossi) [inline application/controllers/Admin.php]: * added "nav-link" class
* Removed Glyphicon

File: application/views/smarty_templates/accounts/view.tpl [hunk 3]
```diff
@@ -91,15 +90,15 @@
                     {/if}
 
                     {if isset($items) && count($items) > 0}
-                    <div class='well' style="background:white">
+                    <div class="card bg-light card-body mb-3" style="background:white">
```
Comment (danfossi) [inline application/views/smarty_templates/accounts/view.tpl]: * class="well" now is  class="card bg-light card-body mb-3"

File: application/views/smarty_templates/footer.tpl [hunk 4]
```diff
@@ -1,7 +1,7 @@
       </div> <!-- /row-fluid -->
       <hr />
       <footer>
-        <p class="pull-right">
+        <p class="float-right">
```
Comment (danfossi) [inline application/views/smarty_templates/footer.tpl]: * class "pull-right" now is "float-right"

File: application/views/smarty_templates/header.tpl [hunk 5]
```diff
@@ -2,80 +2,71 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
     <meta name="description" content="{$header.site_description|escape:'html':'UTF-8'}" />
     <title>{$header.title|escape:'html':'UTF-8'} | {$header.site_title|escape:'html':'UTF-8'}</title>
     <link rel="stylesheet" type="text/css"  href="{url type="base_url" url=""}assets/css/bootstrap.css">
-    <link rel="stylesheet" type="text/css" href="{url type="base_url" url=""}assets/css/style.css">
     <!-- JavaScript -->
-    <script src="{url type="site" url="assets/js/jquery-1.8.1.min.js"}"></script>
+    <script src="{url type="site" url="assets/js/jquery-3.2.1.min.js"}"></script>
```
Comment (danfossi) [inline application/views/smarty_templates/header.tpl]: * JQuery Update

**Comment by danfossi** [comment]
Migration completed!

**Comment by serhack** [comment]
Could I merge this?

**Comment by danfossi** [comment]
Yes, remain to fix just the mobile view (not perfect at the moment)

**Pull request merged**